### PR TITLE
Adds general task to set TES biases

### LINKED ISSUE #544: Add ability turn detector bias off in pysmurf controller
Requested by @kmharrington.

When added, make sure ability propagates to ocs-web.

### PR DESCRIPTION
**Pull request by jlashner**
Adds a general task to set tes biases to specified values, and a separate task to zero biases. Resolves #544.

## Description

Adds a needed function to the pysmurf-controller `set_biases` that lets you generally set tes biases. Also adds `zero_biases` task which can makes it easier to zero all biases from OCS web.

## Motivation and Context

We need more control over TES biases from pysmurf-controller.

## How Has This Been Tested?

Not tested yet.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by kmharrington** [comment]
We could really use these functions running in the field. What's needed to get it going there?

**Review by BrianJKoopman — CHANGES_REQUESTED** [review]
Looks good, mostly docstring related requests.

File: socs/agents/pysmurf_controller/agent.py [hunk 1]
```diff
@@ -852,6 +852,61 @@ def overbias_tes(self, session, params):
 
         return True, "Finished Overbiasing TES"
 
+    @ocs_agent.param('bgs', default=None)
+    @ocs_agent.param('bias')
```
[the remark this answers is not in the document]
Comment (jlashner) [reply socs/agents/pysmurf_controller/agent.py:856]: decided that bias shouldn't be optional for this func, but forgot to change the docstring.

**Comment by jlashner** [comment]
sorry that took so long, but addressed @BrianJKoopman's feedback

**Review by BrianJKoopman — APPROVED** [review]
Thanks for the updates!

**Pull request merged**